src/main/resources/db/migration/V1_56__stilling_fra_nav_status.sql

```diff
@@ -0,0 +1,9 @@
+alter table STILLING_FRA_NAV
+    add LIVSLOPSSTATUS varchar(255) not null default 'HAR_SVART';
+
+update STILLING_FRA_NAV
+set LIVSLOPSSTATUS='PROVER_VARSLING'
+where CV_KAN_DELES is null;
+
+alter table STILLING_FRA_NAV
+    ALTER COLUMN LIVSLOPSSTATUS varchar(255) not null;
```

Comment:
tror man eksplsit må sette default null i oracle

Reply:
Det er ikke det samme, altså default null og not null. Og `ALTER COLUMN LIVSLOPSSTATUS varchar(255) default null not null` gir vel ikke helt mening

Reply:
`ALTER COLUMN LIVSLOPSSTATUS varchar(255) not null default null;`

Reply:
Skjønner. Forsøker (null) varianten. Den ser mest konsistent ut.